test/resolver.spec.js

```diff
@@ -77,19 +78,36 @@ describe('IPLD format resolver (local)', () => {
 
         expect(paths).to.eql([
           'name',
+          'nest',
+          'nest/foo',
           'nest/foo/bar',
+          'array',
+          'array/0',
           'array/0/a',
           'array/1',
-          'someLink///0',
```

Comment:
Ah, yes, but this is the output of .tree test, not the resolve :) 

You added a patch to enable those cases :)